### PR TITLE
add dry option; return useful config

### PR DESCRIPTION
**Pull request by krehermann**
this allows us to dry run and get useful configuration of ocr3 contract changes

### CODE REVIEW

File: deployment/keystone/deploy.go [hunk 1]
```diff
@@ -423,45 +423,66 @@ func ConfigureOCR3Contract(env *deployment.Environment, chainSel uint64, dons []
...
+	OCR3Config *OracleConfigWithSecrets
+	DryRun     bool
+}
+
+func ConfigureOCR3ContractFromJD(env *deployment.Environment, cfg ConfigureOCR3Config) (*ConfigureOCR3Resp, error) {
+	prefix := ""
+	if cfg.DryRun {
+		prefix = "DRY RUN: "
+	}
+	env.Logger.Infof("%sconfiguring OCR3 contract for chain %d", cfg.ChainSel, prefix)
```
Comment (bolekk) [inline deployment/keystone/deploy.go]: wrong order

File: deployment/keystone/deploy.go [hunk 2]
```diff
@@ -423,45 +423,66 @@ func ConfigureOCR3Contract(env *deployment.Environment, chainSel uint64, dons []
...
+	//func ConfigureOCR3ContractFromJD(env *deployment.Environment, chainSel uint64, nodeIDs []string, addrBook deployment.AddressBook, cfg *OracleConfigWithSecrets) (*ConfigureOCR3Resp, error) {
```
Comment (bolekk) [inline deployment/keystone/deploy.go]: delete?

File: deployment/keystone/deploy.go [hunk 3]
```diff
@@ -423,45 +423,65 @@ func ConfigureOCR3Contract(env *deployment.Environment, chainSel uint64, dons []
 	return nil
 }
 
-func ConfigureOCR3ContractFromJD(env *deployment.Environment, chainSel uint64, nodeIDs []string, addrBook deployment.AddressBook, cfg *OracleConfigWithSecrets) error {
-	registryChain, ok := env.Chains[chainSel]
+type ConfigureOCR3Resp struct {
+	Orc2drOracleConfig
```
Comment (patrickhuie19) [inline deployment/keystone/deploy.go]: what does this name mean? 

//nit could you leave a comment or change to a more intuitive name?

**Review by patrickhuie19 — DISMISSED** [review]

File: deployment/keystone/deploy.go (hunk 3 above)
Comment (vyzaldysanchez) [reply deployment/keystone/deploy.go]: I guess this means OCR2 Dry Run Oracle Config?

**Review by bolekk — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**